pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -101,32 +102,25 @@ func (r *ReconcileCheInstallation) Reconcile(request reconcile.Request) (reconci
 		return reconcile.Result{}, err
 	}
 
-	err = r.ensureCheInstallation(reqLogger, cheInstallation)
-	return reconcile.Result{}, err
-}
-
-func (r *ReconcileCheInstallation) ensureCheInstallation(logger logr.Logger, cheInstallation *v1alpha1.CheInstallation) error {
-	ns := cheInstallation.Spec.CheOperatorSpec.Namespace
-
-	if created, err := r.ensureCheNamespace(logger, cheInstallation); err != nil {
-		return r.wrapErrorWithStatusUpdate(logger, cheInstallation, r.setStatusCheSubscriptionFailed, err, "failed to create namespace %s", ns)
-	} else if created {
-		return nil
+	if requeue, err := r.ensureCheNamespace(reqLogger, cheInstallation); err != nil {
+		return reconcile.Result{}, r.wrapErrorWithStatusUpdate(reqLogger, cheInstallation, r.setStatusCheSubscriptionFailed, err, "failed to create namespace %s", cheInstallation.Spec.CheOperatorSpec.Namespace)
+	} else if requeue {
+		return reconcile.Result{Requeue: true, RequeueAfter: 3 * time.Second}, nil
```

Comment:
@MatousJobanek well, I thought it would be needed if we want to requeue only after a certain amount of time. Otherwise, what we got was 10 reconcile calls and then a timeout on the test, hence a failure, whereas with the `RequeueAfter: 3 * time.Second` argument, the test passed.

Reply:
ok, then keep it as it is. I believe that an additonal reconcile will not hurt us :-)